metricbeat/module/system/raid/raid.go

```diff
@@ -67,33 +69,67 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 		return nil, err
 	}
 
+	sysMountPoint := filepath.Join(config.MountPoint, "/sys/block")
+
 	return &MetricSet{
 		BaseMetricSet: base,
 		fs:            fs,
+		sysfs:         sysMountPoint,
 	}, nil
 }
 
+//Annoyingly, different linux subsystems report size size using different blocks
+// /proc/mdstat /proc/partitions and mdadm (Via the BLKGETSIZE64 ioctl call) count "size" as the number of 1024-byte blocks.
+// /sys/block/md*/size uses 512-byte blocks. As does /sys/block/md*/md/sync_completed
+//convert the 512-byte blocks to 1024 byte blocks to maintain how this metricset "used to" report size
+func blockto1024(b int64) int64 {
+	return b / 2
+}
```

Comment:
Please add a note about this to the documentation of the affected fields. Or maybe we could just use bytes as unit for them.